README.md

```diff
@@ -1 +1,174 @@
-# microverse-module-4-ruby-group-capstone
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Catalog of My Things - Ruby Group Capstone ](#-catalog-of-my-things---ruby-group-capstone-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
```

Comment:
- [ ] I understand that sometimes it's hard to agree upon a time as a group of 3 people. However, you've done so great so far, please rearrange a team meeting to record the maximum 5-mins presentation video in order to get the approval of this great app. 🙏💯
_Don't forget to attach the link to the presentation video here in the README file._